### ISSUE TITLE
Website

### ISSUE THREAD

**Issue by yeqbfgxjiq** [1Hive/hcl-mvp#4]
The portal for people to learn about and interact with all things HCL related
- [ ] landing page with high level description and the HCL registry
- [ ] user guides explaining how to use the HCL registry and how to register assets
- [ ] wiki explaining core concepts
- [ ] contributors section for people who want to get involved and hack on the HCL Registry
- [ ] links to all our socials and whatnot

**Comment by lkngtn** [comment]
I have hivecommons.org (currently just redirects 1hive.org) but if we want to have a portal that is focused specifically on the license we can use that (I think a good idea as I think the audience may get distracted by all the DAO stuff on 1hive.org)

**Comment by yeqbfgxjiq** [comment]
Yeah that sounds great. Do you think we can get the basic high level stuff drafted this afternoon so that we could have a simple website/landing-page by the end of the weekend?

**Comment by lkngtn** [comment]
My thinking is that we should divert contributors to 1hive, and keep the hivecommons site focused on documentation for users. 

I'm thinking that the eventually the compliance registry/portal would actually be hosted there as well as general information about the license and mission.

**Comment by lkngtn** [comment]
I tend to think that we should have a FAQ for core concepts rather than a wiki. (much easier to maintain and more conducive to linking out for most of the explanation).

**Comment by yeqbfgxjiq** [comment]
I don't know what you're saying here.

I was saying that we need to create a high level overview and explanation of what the HCL is (as outlined in #2) and put that on the internet on a webpage. That way people who interested (users) can learn about and engage with the HCL project. Then we can start integrate the HCL Registry portal lik we talked bout last week in the [ideas repo Issue](https://github.com/1Hive/ideas/issues/14) and on Keybase. This repo was created to start that process (website, docs/faq, and then React UI and contracts for an HCL registry). 

What are you saying when you refer to contributors and users? 
Are you saying that the HCL would be hosted on 1hive.org vs  hivecommons.org? If so, why?
If the Wiki was an FAQ page, would there also be a "Recommended Resources" page or education portal where people can learn more about harberger related things?

**Comment by lkngtn** [comment]
Sorry should have been more clear, my understanding is that issue is to discuss creating a web portal for people to learn about the HCL. 

From an audience perspective we have a couple of people/personas to think about:
- Someone who is causally looking for information about the license, its properties, and its goals
- A developer of an open source project looking to know how to license their project under the license
- An entrepreneur looking to understand how they can use HCL license software and be compliant 

It makes sense for us to provide information for these users on the site, as well as also host the compliance registry itself there. 

We also may have people who are interested in contributing to the project directly (they might be lawyers, they might be devs that can help build out the registry itself). For these people I think we should direct them to 1hive (rather than having a seperate set of contributor guidelines, and project pages/documentation). 

So the distinction is essentially that registry is a 1hive development project and contributors onboarding and dev documentation live there. The hivecommon.org site hosts the actual registry itself and associated user documentation including FAQs about the license. 

Does that make more sense?

**Comment by yeqbfgxjiq** [comment]
Awesome. Yeah that makes total sense. 100% in agreement
